### ISSUE TITLE
[Bug]: /execute route is commented and not working

### ISSUE THREAD

**Issue by matifanger** [coollabsio/coolify#4312]
### Error Message and Logs

Hey! can i know why /execute route is commented? i need it

![Image](https://github.com/user-attachments/assets/db2cdb6d-4a23-48db-bdce-ca966cd72e91)

Also would be good to have access to /execute into databases too

### Steps to Reproduce

1. Just try to /execute, returns 404

### Coolify Version

v4.0.0-beta.369

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubuntu 22.04

**Comment by peaklabs-dev** [comment]
Because we now have a full terminal inside Coolify.

**Comment by matifanger** [comment]
> Because we now have a full terminal inside Coolify.

Yes, but in my case, I have a PostgreSQL server, and I’d like to create multiple databases within it. One way to do this is through the terminal, but I need to automate the process. The /execute endpoint seemed perfect for achieving that — unless there are better alternatives that could work more effectively.

**Comment by matifanger** [comment]
Can we've this up and working again? @peaklabs-dev

**Comment by gorlug** [comment]
I also want to run some automated commands on the database, so I'd really like this back in or an alternative approach.

But what's even worse from an enduser point of view: The route is still in the API docs even though it clearly is not active.
